build.sbt

```diff
@@ -299,6 +300,10 @@ lazy val server = (project in file("server"))
 
       // Auth dependencies
       "com.unboundid.product.scim2" % "scim2-sdk-common" % "3.1.0",
+      "org.casbin" % "jcasbin" % "1.55.0",
+      "org.casbin" % "jdbc-adapter" % "2.7.0"
+        exclude("com.microsoft.sqlserver", "mssql-jdbc")
+        exclude("com.oracle.database.jdbc", "ojdbc6"),
```

Comment:
Partially because of licensing related issues, but I also kind of feel like it makes more sense for a customer to install the drivers themselves if that's the database they choose to use. This is consistent with our instructions for using MySQL.